tools/run_pylint.py

```diff
@@ -7,19 +7,25 @@
 
 BASE_PATH = sys.argv[1]
 
-FILE_NAME = re.compile(r'[-a-zA-Z0-9_/]*\.py')
+FILE_NAME = re.compile(r'[-a-zA-Z0-9_/]*\.py$')
+TEST_FOLDER_NAME = re.compile(r'tests')
 CODE_RATING = re.compile('Your code has been rated at 10')
 
 #NB: If you change it, you must also change it in .travis.yml
-COMMAND = 'pylint src/server/server src/scraper/scraper src/common/common src/topicmodeller/topicmodeller ' \
-          'src/learner/learner src/orchestrator/orchestrator -f parseable'
-TEST_COMMAND = 'pylint --rcfile=pylintrc_tests src/server/tests/*.py src/scraper/tests/*.py src/common/tests/*.py ' \
-               'src/functests/*.py src/topicmodeller/tests/*.py src/learner/tests/*.py src/orchestrator/tests/*.py ' \
-               '-f parseable'
+COMMAND = 'pylint -f parseable'
+TEST_COMMAND = 'pylint --rcfile=pylintrc_tests -f parseable'
+
+
+def files_list(base_folder, for_tests):
+    result = []
+    for root_folder, sub_folders, files in os.walk(base_folder, topdown=False):
+        is_test_folder = TEST_FOLDER_NAME.search(root_folder) is not None
```

Comment:
Why to we need to override default parameter topdown to false (default is true) ?